main.go

```diff
@@ -92,7 +92,6 @@ func getToken(uaaURL, user, secret string) (string, error) {
 	post.Send(params.Encode())
 
 	cmd, _ := post.AsCurlCommand()
```

Comment:
We're nearly there now.  You can also remove line 94 (this will actually fail to compile as `cmd` is unused, and you can't have unused vars in Go).

That line only existed for troubleshooting and never should have made it into a commit.

Reply:
You can't have unused vars in Go? Thats interesting.